recipes/librealsense/all/patches/patch-libusb-include.patch

```diff
@@ -0,0 +1,16 @@
+--- CMake/global_config.cmake
++++ CMake/global_config_new.cmake
+@@ -111,10 +111,10 @@
+ 
+ macro(add_tm2)
+     message(STATUS "Building with TM2")
+-    include(libusb_config)
+-    target_link_libraries(${LRS_TARGET} PRIVATE usb)
++    find_package(libusb REQUIRED)
++    target_link_libraries(${LRS_TARGET} PRIVATE libsusb)
```

Comment:
If this is using a Conan cmake_find_package, then you're looking to a library instead of a library target